### DISCUSSION TITLE
why the same sentence is generated differently on the same model

### DISCUSSION THREAD

**Discussion by pivolan** [coqui-ai/TTS#2799] [General Q&A]
I use VITS and yourTTS, single model trained from zero on 4 hours dataset in dutch language. And every time when I generate audio from sentence, I get a little bit different results
It sounds good, but a little bit different. Why and for what?
![image](https://github.com/coqui-ai/TTS/assets/903550/3b98d1e9-d778-4a09-9265-085ef551c4e8)

**Comment by Bestom927** [comment]
Because of the duration predictor? which Samples a random value from the distribution

**Reply by pivolan** [reply]
Daar heb ik begrip voor. Daar moet ook meer geld naar, maar je kunt dat niet vergelijken.
this is NL(Dutch) language

audio samples
https://pub-8aef3f425d5545cbb79d2622c37b432d.r2.dev/1_tts12_checkpoint_60000_Daar.wav
https://pub-8aef3f425d5545cbb79d2622c37b432d.r2.dev/2_tts12_checkpoint_60000_Daar.wav
https://pub-8aef3f425d5545cbb79d2622c37b432d.r2.dev/3_tts12_checkpoint_60000_Daar.wav
https://pub-8aef3f425d5545cbb79d2622c37b432d.r2.dev/4_tts12_checkpoint_60000_Daar.wav
https://pub-8aef3f425d5545cbb79d2622c37b432d.r2.dev/5_tts12_checkpoint_60000_Daar.wav

**Comment by pivolan** [comment]
sorry, it was not german language, it is dutch.
